pkgs/development/python-modules/reno/default.nix

```diff
@@ -79,6 +85,6 @@ python3Packages.buildPythonApplication rec {
     mainProgram = "reno";
     homepage = "https://docs.openstack.org/reno/latest";
     license = licenses.asl20;
-    maintainers = teams.openstack.members ++ (with maintainers; [ drewrisinger guillaumekoenig ]);
```

Comment:
Why are the maintainers removed? IMHO, removal of maintainers requires more rationale than saying "misc cleanup" in the commit message.